citesphere/src/main/java/edu/asu/diging/citesphere/web/user/authorities/AddAuthorityController.java

```diff
@@ -0,0 +1,55 @@
+package edu.asu.diging.citesphere.web.user.authorities;
+
+import java.util.HashSet;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import edu.asu.diging.citesphere.core.service.IAuthorityService;
+import edu.asu.diging.citesphere.model.authority.IAuthorityEntry;
+import edu.asu.diging.citesphere.model.authority.impl.AuthorityEntry;
+import edu.asu.diging.citesphere.user.IUser;
+import edu.asu.diging.citesphere.web.forms.AuthorityForm;
+
+@Controller
+public class AddAuthorityController {
+
+    @Value("${_authority_uri}")
+    private String authorityUri;
+
+    @Value("${_authority_prefix}")
+    private String authorityPrefix;
+
+    @Autowired
+    private IAuthorityService authorityService;
+
+    @GetMapping(value = "/auth/authority/new")
+    public String show(Model model) {
+        model.addAttribute("authorityForm", new AuthorityForm());
+        return "auth/authorities/new";
+    }
+
+    @PostMapping(value = "/auth/authority/new")
```

Comment:
let's call that `create` instead of `new` (or whatever term is used for citations).

Reply:
It is "add" for others, but there's already an "/auth/authority/add" POST api, so I used "new" to avoid confusion

Reply:
there is a method for `/auth/authority/create` but it actually imports an authority; let's change the existing one to `/auth/authority/import` and then make this one `/auth/authority/create`. it's more accurate.